### PR TITLE
C# implementation

### PR DESCRIPTION
**Pull request by nathan-contino-mongo**
- switched from old version of driver to new version
- added some helpful commands so others can run the project more easily/understand dependencies
- actually works

Haven't "normalized" this to match the rest of your repo but it's a start. Let me know what looks unorthodox to you and I'll work on it.

### CODE REVIEW

**Review by ccho-mongodb — APPROVED** [review]

**Comment by nathan-contino-mongo** [comment]
@ccho-mongodb I can't merge this in, feel free to merge when ready

**Pull request merged**
